lib/mozart_fetcher/http_client.ex

```diff
@@ -37,7 +37,11 @@ defmodule HTTPClient do
   end
 
   defp make_request(endpoint, headers, options, client) do
-    client.get(endpoint, headers, options)
+    client.get(endpoint, request_headers(headers), options)
+  end
+
+  defp request_headers(headers) do
+    headers ++ [{ 'User-Agent', 'Mozart-Fetcher' }]
```

Comment:
Can this be "MozartFetcher" as in the ticket.

Reply:
Apologies will update that now :)

Reply:
Its cause the ticket title says 'Mozart-Fetcher' but you are right